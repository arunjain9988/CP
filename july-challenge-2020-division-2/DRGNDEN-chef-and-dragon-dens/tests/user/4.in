10 10
9 8 7 6 5 4 3 2 1 10
10 11000 23 6 5 6 8 9 6 52
2 2 10
2 1 10
2 4 8
2 8 8
2 8 7
8 9 4
2 5 1
2 1 1
2 10 10
2 10 8
