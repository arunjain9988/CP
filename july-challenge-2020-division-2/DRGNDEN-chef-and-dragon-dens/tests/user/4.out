-1
-1
34
9
-1
-1
-1
10
52
61